ubuntu/14.04/standard/Dockerfile

```diff
@@ -71,6 +72,14 @@ RUN echo "deb [arch=amd64] http://apt-mo.trafficmanager.net/repos/dotnet-release
     dotnet-dev-1.0.1 \
  && rm -rf /var/lib/apt/lists/*
 
+# Run dotnet command to populate initial cache
+RUN dotnet help
+
+# Accept the TEE EULA
+RUN mkdir -p "/root/.microsoft/Team Foundation/4.0/Configuration/TEE-Mementos" \
```

Comment:
Actually, we probably want to add this step to the base images. Without it, anybody trying to use TFVC (on VSTS or TFS) instead of git will hit an issue for not accepting the license.